### PR TITLE
COM-5525 : B2B NonPurchaser can initiate Quotes

### PR DESCRIPTION
**Pull request by kibo-pawanjadhav**

### CODE REVIEW

File: scripts/modules/b2b-account/quotes.js [hunk 1]
```diff
@@ -126,6 +126,9 @@ define([
         },
         render: function () {
             var self = this;
+            var isNonPurchaserUser=  !require.mozuData('user').behaviors.includes(1000) && !require.mozuData('user').behaviors.includes(1005);
```
Comment (kibo-raghu) [inline scripts/modules/b2b-account/quotes.js]: Add space before `=`

File: scripts/modules/b2b-account/quotes.js [hunk 2]
```diff
@@ -136,6 +139,11 @@ define([
             }
             else {
                 collection = new QuotesGridCollectionModel({ autoload: true });
+                console.log(collection);
+                if(isNonPurchaserUser)
+                {
+                    collection.attributes.rowActions=[];
```
Comment (kibo-raghu) [inline scripts/modules/b2b-account/quotes.js]: Add space before and after `=`

File: scripts/modules/b2b-account/quotes.js [hunk 3]
```diff
@@ -136,6 +139,11 @@ define([
             }
             else {
                 collection = new QuotesGridCollectionModel({ autoload: true });
+                console.log(collection);
```
Comment (kibo-raghu) [inline scripts/modules/b2b-account/quotes.js]: Remove Console.Log everywhere

File: scripts/modules/b2b-account/quotes.js [hunk 4]
```diff
@@ -416,13 +424,14 @@ define([
                 sortable: false
             }
         ],
-        rowActions: [
+        rowActions:
```
Comment (kibo-raghu) [inline scripts/modules/b2b-account/quotes.js]: Revert changes if nothing is added

File: scripts/modules/b2b-account/view-quote.js [hunk 5]
```diff
@@ -39,6 +39,10 @@ define([
             self.model.set("isUserAdmin", require.mozuData('user').behaviors.includes(1000));
             self.model.set("isUserPurchaser", require.mozuData('user').behaviors.includes(1005));
             self.model.set('isSalesRep', require.mozuData('user').isSalesRep);
+            self.model.set("isUserPurchaser", require.mozuData('user').behaviors.includes(1005));
+            var isNonPurchaserUser=  !require.mozuData('user').behaviors.includes(1000) && !require.mozuData('user').behaviors.includes(1005);
+            self.model.set("isNonPurchaserUser", isNonPurchaserUser);
+            console.log(self.model.get("isNonPurchaserUser"));
```
Comment (kibo-raghu) [inline scripts/modules/b2b-account/view-quote.js]: Remove console.log

File: scripts/modules/b2b-account/view-quote.js [hunk 6]
```diff
@@ -39,6 +39,10 @@ define([
             self.model.set("isUserAdmin", require.mozuData('user').behaviors.includes(1000));
             self.model.set("isUserPurchaser", require.mozuData('user').behaviors.includes(1005));
             self.model.set('isSalesRep', require.mozuData('user').isSalesRep);
+            self.model.set("isUserPurchaser", require.mozuData('user').behaviors.includes(1005));
+            var isNonPurchaserUser=  !require.mozuData('user').behaviors.includes(1000) && !require.mozuData('user').behaviors.includes(1005);
```
Comment (kibo-raghu) [inline scripts/modules/b2b-account/view-quote.js]: Add spaces before after `=` everywhere

File: scripts/modules/cart-common-view.js [hunk 7]
```diff
@@ -62,16 +62,18 @@ define(['modules/api',
...
-            //       discount.getDiscountDetails().then(function(){
-
-            //       })
-            //    })
-            //})
-        },
+        }, 
+        userHasBehavior: function (behaviorId) {
+          var behaviors = require.mozuData('user').behaviors;
+          console.log( require.mozuData('user'));
```
Comment (kibo-raghu) [inline scripts/modules/cart-common-view.js]: remove console.log

**Review by kibo-raghu — CHANGES_REQUESTED** [review]

**Review by Kibo-Sagar — APPROVED** [review]
changes looks good to me.

**Review by kibo-raghu — APPROVED** [review]

**Pull request merged**